Source/DafnyDriver/DafnyDriver.cs

```diff
@@ -169,6 +169,23 @@ public enum CommandLineArgumentsResult {
       OK_EXIT_EARLY
     }
 
+    static HashSet<string> SplitOptionValueIntoFiles(HashSet<string> inputs) {
+      var output = new HashSet<string>();
+      foreach (var input in inputs) {
+        var values = input.Split(',');
+        foreach (var slice in values) {
+          var name = slice.Trim();
+          if (System.IO.Directory.Exists(name)) {
```

Comment:
I would leave out the `System.IO` in front of Directory.

Reply:
Edited in a followup PR.